applications/serial_lte_modem/src/slm_at_udp_proxy.c

```diff
@@ -663,3 +679,36 @@ int slm_at_udp_proxy_uninit(void)
 
 	return 0;
 }
+
+/**@brief API to get datamode from external
+ */
+bool slm_udp_get_datamode(void)
+{
+	return udp_datamode;
+}
+
+/**@brief API to set datamode from external
+ */
+void slm_udp_set_datamode_off(void)
+{
+	udp_datamode = false;
+
+	if (udp_sock != INVALID_SOCKET) {
+		if (udp_server_role) {
+			sprintf(rsp_buf, "#XUDPSVR: 0, \"datamode\"\r\n");
```

Comment:
This URC does not sync with TCP proxy server. In TCP proxy server there is no "datamode" URC.

Reply:
Removed, customer don't need this URC